Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIStorageCocoa.mm

```diff
@@ -64,6 +64,19 @@
     }).get()];
 }
 
+void WebExtensionContext::storageGetKeys(WebPageProxyIdentifier webPageProxyIdentifier, WebExtensionDataType dataType, CompletionHandler<void(Expected<Vector<String>, WebExtensionError>&&)>&& completionHandler)
+{
+    auto callingAPIName = makeString("browser.storage."_s, toAPIString(dataType), ".getKeys()"_s);
+
+    auto storage = storageForType(dataType);
```

Comment:
Should be a pointer too.
```suggestion
    auto *storage = storageForType(dataType);
```

Reply:
Sounds good. I'll update the few others in the file as well.